src/LibraryManager.Vsix/Contracts/HostInteraction.cs

```diff
@@ -109,7 +109,7 @@ public Task<Stream> ReadFileAsync(string filePath, CancellationToken cancellatio
 
         public Task<bool> CopyFile(string sourcePath, string destinationPath, CancellationToken cancellationToken)
         {
-            return System.Threading.Tasks.Task.Run(() =>
+            return Task.Run(() =>
```

Comment:
Just so I'm clear, the File.Copy call is faster than using the actual async IO operations? IE, something like:

public async Task<bool> CopyFile(string sourcePath, string destinationPath, CancellationToken cancellationToken)
{
	bool success = true;

	try
	{
		using (FileStream sourceStream = File.Open(sourcePath, FileMode.Open))
		{
			using (FileStream destinationStream = File.Create(destinationPath))
			{
				await sourceStream.CopyToAsync(destinationStream, cancellationToken);
			}
		}
	}
	catch
	{
		success = false;
	}

	return success;
}

is slower?

Reply:
That's the gist of this PR (through a few abstraction layers), yes.

Reply:
The old code was slightly different though. It was doing quite a bit differently from this. Could you do me a favor and verify that the code I added above is also slower than your change?

Reply:
Based on our conversation and further diligence, you're right they're approximately equal.  I'll rewrite `CopyFile` => `CopyFileAsync` and make it actually async to match the rest of the file operations here.